elastic/kibana
# [Stack mgt apps] Wrap all apps with KibanaThemeProvider

This PR wraps all the apps of the Stack management team with the new `<KibanaThemeProvider/>`.

## Plugins

* `license_managment`
* `cross_cluster_replication`
* `watcher`
* `remote_clusters`
* `rollup`
* `snapshot_restore`
* `ingest_pipelines`
* `index_management`
* `index_lifecycle`
* `upgrade_assistant`
* `dev_tools`
* `console`
* `searchprofiler`
* `grokdebugger`
* `painless_lab`
* `management`

**Note**: there is one commit for each plugin change

## How to test

There should not be much manual test to do as we should have smoke tests for all of our apps. Navigate to each of the app and verify that it renders correctly.

See https://github.com/elastic/kibana/issues/118866 for context.
